src/constants.ts

```diff
@@ -86,3 +86,9 @@ export const PROCESS_INPUT = {
   ROGUELIKE_RESUME_ADVENTURE: 100011,
   CREATE_REPLAY: 100012
 };
+
+// Default shortcut inputs
+export const DEFAULT_SHORTCUTS = {
```

Comment:
We could use key codes instead so that there is only one value rather than an Array e.g. m is keycode 77.

keyCode is a property on the keydown event.

Reply:
[keyCode is deprecated](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/keyCode) but we could use `KeyBoardEvent.code`

Reply:
Thank you for your feedback!
I've made the modifications in the following commit 😸